src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -320,6 +342,13 @@ public string ToDisplayString(SymbolDisplayFormat format = null)
             return str;
         }
 
+        private bool IsSafeToResolve()
+        {
+            var declaringMethod = (DefaultType as TypeParameterSymbol)?.DeclaringMethod as SourceOrdinaryMethodSymbol;
+            return !((object)declaringMethod != null && !declaringMethod.HasComplete(CompletionPart.FinishMethodChecks) &&
```

Comment:
>return [](start = 12, length = 6)

nit: may be easier  to read if inverted: `return declaringMethod is null || declaringMethod.HasComplete(FinishMethodChecks) || (!declaringMethod.IsOverride && !declaringMethod.IsExplicitInterfaceImplementation)`.

Reply:
> may be easier  to read if inverted

I intentionally used this way because it looks easier to understand for me.

---
In reply to: [276460678](https://github.com/dotnet/roslyn/pull/35048#discussion_r276460678) [](ancestors = 276460678)